Azure-Samples/observability-accelerators
# Update ACR Permissions

## Purpose

- Fix execute permission on kubelogin.sh
- Update role definition in bicep deployment

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[X] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
* Clone the code in WSL or on Linux and validate that the dev container builds (tests the `kubelogin.sh` execute flag change) 
* Deploy via bicep and inspect the role assignments on the container registry. Prior to this change, the cluster identity is assigned `Contributor`, after the change it should be `AcrPull`

## Other Information
